ibm-messaging-mq-cloud-showcase-app/backend/AWS-showcase-backend/models/Producer.js

```diff
@@ -18,15 +18,16 @@ const MQClient = require("../mqclient/mqclient");
 
 class Producer {
     constructor() {        
-        this.mqclient = new MQClient();
+        this.mqclient = new MQClient();        
     }
 
-    putMessages(message, quantity, queueName) {
+    putMessages(message, quantity, queueName, currency) {
```

Comment:
4 params is about the limit where they might be better of sending as a class instance or a json object.